src/main/java/duke/ui/Ui.java

```diff
@@ -0,0 +1,123 @@
+package duke.ui;
+
+import duke.task.Task;
+import duke.task.TaskList;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ * The user interaction reactor.
+ */
+public class Ui {
+    private Scanner scanner = new Scanner(System.in); //The scanner scans user command.
+
+    /**
+     * Prints welcome message.
+     */
+    public void showWelcome() {
+        System.out.println("____________________________________________________________");
```

Comment:
In the **UI** class, maybe you could assign the line to be printed to a constant? Cuz this would make it easier for you to change your code if there's a change in the requirement needed. This can be done for the indentation too, i.e. make a constant indentation string and append to the sentences that need indentation so that you can change the indentations altogether easily if you need